its/src/test/java/com/sonar/it/scanner/msbuild/utils/ProvisioningAssertions.java

```diff
@@ -57,8 +57,8 @@ public static void assertCacheMissBeginStep(BuildResult begin, String sqApiUrl,
     assertThat(beginLogs).contains(
       "JreResolver: Resolving JRE path.",
       "Downloading from " + sqApiUrl + "/analysis/jres?os=" + os + "&arch=" + arch + "...",
-      "Response received from " + sqApiUrl + "/analysis/jres?os=" + os + "&arch=" + arch + "...",
-      "Cache miss. Attempting to download JRE.");
+      "Response received from " + sqApiUrl + "/analysis/jres?os=" + os + "&arch=" + arch + "...");
+    TestUtils.matchesSingleLine(beginLogs, "Cache miss. Attempting to download '" + cacheFolderPattern + "'");
```

Comment:
`Attempting to download '/somelocalFile.zip'` is a bit confusing. It should be `Attempting to download to '/somelocalFile.zip'` instead.